src/static/helpers/slasHelper.test.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
Why is this added?

Reply:
We changed some things so the default test environment is jsdom and certain tests fail because the test environment is not node